docs/content/docs/data-structures/sequence.md

```diff
@@ -0,0 +1,241 @@
+---
+title: Sequences and Strings
+menuPosition: 3
+---
+
+<!-- IMPORTANT: Don't manually edit the auto-generated content below. Instead, edit the source file (look at the path
+     argument) and then run "npm run readme:update" from the root of the repo. That will update all autogenerated
+     content, so you should commit the resulting changes. -->
+
+<!-- AUTO-GENERATED-CONTENT:START (INCLUDE_ROOT:path=packages/dds/sequence/README.md&start=2) -->
+Every item in a SharedSegmentSequence is at a specific position starting at 0, kind of like an array. However, it
+differs from an array in that the positions can move as local and remote collaborators make modifications to the
+sequence. There are a number of different sequence types:
+
+- SharedString for storing and collaborating on a sequence of text.
+- SharedNumberSequence for storing and collaborating on a sequence of numbers.
+- SharedObjectSequence for storing and collaborating on a sequence of JSON-serializable objects.
+
+As the name suggests SharedSegmentSequence, or sequence for short, are made of segments. Segments are the leaf nodes of
+the tree data structure that enables collaboration and backs the sequence. Segments may be split and merged as
+modifications are made to the sequence. Every segment has a length from 1, to the length of the sequence. The length of
```

Comment:
"Every segment has a length from 1", or "every segment is counted to be an single element, and length of sequence is number of elements"? The former suggests that it can be not 1 and needs more explanation.

Reply:
A SharedSequence is an array-like ordered sequence of items.  SharedSequence operations are defined in terms of the item's position in the sequence (e.g., removeRange(start, end)).

For efficiency, a SharedSequence stores contiguous blocks of positions/items in segments.  This is mostly an under-the-covers detail.  However, we do expose the ability to enumerate the segments that intersect a range of positions.  Enumerating the segments allows a savvy renderer to avoid per-character tree lookups and string allocations as it builds up the HTML text content.

The intent of the above prose is to communicate that:

a) Segments contain 1 or more positions (i.e., have a minimum length of 1).
b) Segments may be arbitrarily split/merged by the SharedSequence.  Users should generally not retain references to segments outside of the enumeration or have expectations about segment length.

Reply:
I'd rephrase it. I'd say every segment has some length, and sum of segment lengths is the length of sequence.